src/guc.c

```diff
@@ -6,6 +6,8 @@
 bool		guc_disable_optimizations = false;
 bool		guc_optimize_non_hypertables = false;
 bool		guc_restoring = false;
+bool		guc_originating_node = false;
```

Comment:
Could `guc_originating_node` and `guc_ignore_ddl` be the same flag ? Wasn't `guc_originating_node` used to not run DDL commands ?